### ISSUE TITLE
A new stable v0.8 release?

### ISSUE THREAD

**Issue by NobodyXu** [ParkMyCar/compact_str#380]
It has been 8 months since the 0.8.0-beta release on compact-str.

I think it's time to have.a stable release given that there's no comlaint, cc @ParkMyCar can we have a stable new release please?

**Comment by ParkMyCar** [comment]
Good idea, we're long overdue for one. I retained the beta tag originally waiting for https://github.com/ParkMyCar/compact_str/issues/304 to get resolved, but there hasn't been any movement so we'll go without it

**Comment by NobodyXu** [comment]
BTW @ParkMyCar I can volunteer to become maintainer of this repository to review MR and cut new release if you are busy.

**Comment by linkmauve** [comment]
Hi, you can close this issue, as 0.8 and 0.9 got released.

**Comment by NobodyXu** [comment]
Thanks I've closed it
